primary-nav-items.html

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
This file is a duplicate of `_includes/templates/nav/_vars-primary-nav.html`. We probably only need one right?